robyn/__init__.py

```diff
@@ -211,14 +212,16 @@ def start(self, host: str = "127.0.0.1", port: int = 8080, _check_port: bool = T
         port = int(os.getenv("ROBYN_PORT", port))
         open_browser = bool(os.getenv("ROBYN_BROWSER_OPEN", self.config.open_browser))
 
-        if _check_port:
```

Comment:
this is important, for the tests to pass. Otherwise all the integration tests will fail due to timeouts.